src/mkindex_algo.hpp

```diff
@@ -236,7 +234,13 @@ translateSeqs(TCDStringSet<std::vector<TOrigAlph>> & in,
     double start = sysTime();
     myPrint(options, 1, "Translating Subj Sequences...");
 
-    out = in | seqan3::view::translate | std::view::join; //TODO geneticCode
+    auto tmp  = in | seqan3::view::translate //TODO geneticCode
+                   | std::view::join
+                   | std::view::transform([] (auto && elem)
+                     {
+                         return std::forward<decltype(elem)>(elem) | std::ranges::to<std::vector>;
+                     })
+                   | std::ranges::to<decltype(out)>;
```

Comment:
Are the two views after ```view_join``` used in order to make it comparable to later seqan2 types?

Reply:
No, they convert from view to range (which is quite cumbersome for two-dimensional ranges).